src/components/Cardlist/Card/index.js

```diff
@@ -49,26 +50,27 @@ const Card = (props) => {
     }
 
     return (
-      <div className={cardClass}>
-          <p>
-            <CardHeader
-              value={cachedState.cachedCard.caption}
-              handler={changeHandler}
-              caption={props.data.caption}
-              edit={props.data.edited} />
-            <span className="iconBox">
-              {props.data.edited ? <span onClick={saveEditing}><IoIosSave /></span>
-              : (props.checkboxMain ? null : <span onClick={editing}><AiOutlineEdit /></span>)}
-              {props.data.edited ? <span onClick={cancelEditing}><MdCancel /></span>
-              : <input type="checkbox" checked={props.data.checked} onChange={() => props.onCheckboxChange(props.data.id)} />}
-            </span>
-          </p>
-          <CardBody
-            value={cachedState.cachedCard.text}
+      <WithLoadingDelay classes={cardClass}>
```

Comment:
HOCs that add functionality should better wrap your component on export (unless it's a special service component that requires usage in jsx code like context or router providers):
```
...
export default withLoadingDelay(Card);
```